bundles/org.openhab.binding.paradoxalarm/README.md

```diff
@@ -98,14 +98,14 @@ Currently binding supports the following panels: EVO192, EVO48(not tested), EVO9
     String partition1AdditionalStates “Floor1 MUC additional states: [%s]” (Partitions) { channel = “paradoxalarm:partition:ip150:partition1:additionalStates” }
 
 //ZONES
-    Contact CorridorFl1_PIR_state “Corridor Fl1 motion: [%s]” (PIRSensors) { channel = “paradoxalarm:zone:ip150:MotionSensor1:opened” }
-    Contact CorridorFl1_MUC_state “Corridor Fl1 window: [%s]” (Floor1MUC) { channel = “paradoxalarm:zone:ip150:MagneticSensorWindow1:opened” }
+    Contact CorridorFl1_PIR_state “Corridor Fl1 motion: [%s]” (PIRSensors) { channel = “paradoxalarm:zone:ip150:MotionSensor1:isOpened” }

```

Comment:
The channel id is "opened": https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.paradoxalarm/src/main/resources/ESH-INF/thing/zone.xml#L16

Reply:
Well, I first copied the example containing 'opened' and that didn't work. Then I found [1] on the forum, which mentions 'isOpened' instead and that did work.

Disclaimer: I'm new to OpenHAB, so I may be misinterpreting something.

[1] https://community.openhab.org/t/paradox-evo-binding/58026/97

Reply:
It was updated during the review process. The info on the forum is out of date.

Reply:
ok, it all makes sense now:

I'm running org.openhab.binding.paradoxalarm-2.4.0-SNAPSHOT.jar which I pulled from [2]

I've just inspected the zone.xml in my installation and that mentions 'isOpened' instead of 'opened'.

I decided to install 2.4.0-SNAPSHOT, because I'm running OH2.4. Not sure whether 2.5.0-SNAPSHOT would work, wanted to avoid compatibility issues with that...

[2] https://github.com/theater/binaries/tree/master/ParadoxAlarm

Reply:
The documentation in the repo must be relative to the code in the repo. The code, which is 2.5 version in the repo, is using "opened" and the documentation is OK.